### PR TITLE
Biogen datasource - nferx plugin

### PR DESCRIPTION
**Pull request by PriyankaW**
New plugin for nferx. In order to activate this new Biogen specific Nferx plugin, point the 'textMining' evidence section to 'ot-text-mining-nferx' plugin

### CODE REVIEW

**Comment by opentargets-fern** [comment]
Deploy preview ready!

Built with commit c2595ded24e9395cfd56403259311988a021031f

https://deploy-preview-201--paratrooper-antelope-70721.netlify.com

File: app/plugins/text-mining/text-mining.html [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+            <p>Source: Nferx
+                <ot-nferx
+                        target="{{target.id}}"
+                        disease="{{disease.efo}}"
+                        ext="ext.nferx">
+                </ot-nferx>
+            </p>
+        </div>
+    </div>
+
```
Comment (PriyankaW) [inline app/plugins/text-mining/text-mining.html]: @emepyc @LucaFumis @elipapa -There is a separate directive for nferx - nferx-directive.js. But I call this directive from the text-mining html- do you think we should use with this approach? Or  have a new 'Nferx' plugin that includes both the text mining and nferx related directives - it stays part of public repo . In the public version -evidenceSection will point to original text-mining plugin and in the Biogen private version - evidenceSection will override it with the new Nferx plugin?

File: app/src/services/consts-service.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ angular.module('otServices')
                 CHEMBL: 'chembl',
                 DISGENET: 'disgenet',
                 EPMC: 'europepmc',
+                NFERX: 'nferx',
```
Comment (emepyc) [inline app/src/services/consts-service.js]: Luca is already migrating this service to a config file so you will have to create your version (as you are currently doing with default.json vs custom.json)

File: app/src/services/dictionary-service.js [hunk 3]
```diff
@@ -63,6 +63,7 @@ angular.module('otServices')
             // N
             NA: 'N/A',
             NO_DATA: 'No data',
+            NFERX :'Literature - NFERX',
```
Comment (emepyc) [inline app/src/services/dictionary-service.js]: same here... we are moving this to a config json, but you will need to keep it here until it is done

**Pull request closed without merging**